src/mavlink-router/ulog.cpp

```diff
@@ -168,6 +168,7 @@ int ULog::write_msg(const struct buffer *buffer)
         mavlink_msg_logging_ack_encode(_system_id, MAV_COMP_ID_ALL, &msg, &ack);
         _send_msg(&msg, _target_system_id);
         /* no break needed, message will be handled by MAVLINK_MSG_ID_LOGGING_DATA case */
+        _fall_through_;
```

Comment:
a simple `/* fall-through */` should solve this, no? You can even replace the "no break needed" part above